accord-core/src/main/java/accord/local/CommandStore.java

```diff
@@ -43,16 +45,113 @@ CommandStore create(int id,
                             RangesForEpochHolder rangesForEpoch);
     }
 
-    int id();
-    Agent agent();
-    AsyncChain<Void> execute(PreLoadContext context, Consumer<? super SafeCommandStore> consumer);
-    <T> AsyncChain<T> submit(PreLoadContext context, Function<? super SafeCommandStore, T> apply);
+    @Nullable
+    public static CommandStore maybeCurrent()
```

Comment:
[Preference]: I would prefer not to clutter the preamble with these methods, and would stick them at the bottom of the class.